common/browser_context.go

```diff
@@ -129,6 +138,20 @@ func (b *BrowserContext) parseObjectInitScript(script goja.Value) (string, error
 	return "", errors.New("content or path properties must be set for script as object")
 }
 
+func (b *BrowserContext) parseFuncInitScript(script goja.Value, arg goja.Value,
+	isCallable, hasArgs bool,
+) (string, error) {
+	if isCallable && hasArgs {
+		input, err := json.Marshal(arg.Export())
+		if err != nil {
+			return "", fmt.Errorf("can't marshal arg: %w", err)
+		}
+		return fmt.Sprintf("(%s)(%s);", script.ToString().String(), string(input)), nil
```

Comment:
`ToString()` is enough here:

```suggestion
		return fmt.Sprintf("(%s)(%s);", script.ToString(), string(input)), nil
```